database/src/migrations/template_methodology_species_transformations/moose_recruitment_survey.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
question:
is this supposed to be empty?

Reply:
yeah, I just hadn't looked into it yet. Could do that before this gets merged tho, or I can remove it from the current migration and add it after.  Either way.

Reply:
whichever you think is better :)

Reply:
Ill see if I can get it in today.  Probably the least amount of work that way